src/main/java/org/broadinstitute/hellbender/tools/sv/SVDepthOnlyCallDefragmenter.java

```diff
@@ -0,0 +1,120 @@
+
+    public SVDepthOnlyCallDefragmenter(final SAMSequenceDictionary dictionary) {
+        this(dictionary, 0.9);
+    }
+
+    //for single-sample clustering case
+    public SVDepthOnlyCallDefragmenter(final SAMSequenceDictionary dictionary, double minSampleOverlap) {
+        super(dictionary, CLUSTERING_TYPE.SINGLE_LINKAGE);
+        this.minSampleOverlap = minSampleOverlap;
+    }
+
+    /**
+     * Find a single call representative of all the calls in the {@param cluster}
+     * @param cluster   the events that are clustered together
+     * @return  a call encompassing all the cluster's events and containing all the algorithms and genotypes
+     */
+    @Override
+    protected SVCallRecordWithEvidence flattenCluster(final Collection<SVCallRecordWithEvidence> cluster) {
+        final int newStart =  cluster.stream().mapToInt(SVCallRecordWithEvidence::getStart).min().getAsInt();
+        final int newEnd = cluster.stream().mapToInt(SVCallRecordWithEvidence::getEnd).max().getAsInt();
+        final SVCallRecordWithEvidence exampleCall = cluster.iterator().next();
+        final int length = newEnd - newStart + 1;  //+1 because GATK intervals are inclusive
+        final List<String> algorithms = cluster.stream().flatMap(v -> v.getAlgorithms().stream()).distinct().collect(Collectors.toList()); //should be depth only
+        final List<Genotype> clusterGenotypes = cluster.stream().flatMap(v -> v.getGenotypes().stream()).collect(Collectors.toList());
+        return new SVCallRecordWithEvidence(exampleCall.getContig(), newStart, exampleCall.getStartStrand(),
+                exampleCall.getEndContig(), newEnd, exampleCall.getEndStrand(), exampleCall.getType(), length, algorithms, clusterGenotypes,
+                exampleCall.getStartSplitReadSites(), exampleCall.getEndSplitReadSites(), exampleCall.getDiscordantPairs());
+    }
+
+    /**
+     * Determine if two calls should cluster based on their padded intervals and genotyped samples
+     * @param a
+     * @param b
+     * @return true if the two calls should be in the same cluster
+     */
+    @Override
+    protected boolean clusterTogether(final SVCallRecordWithEvidence a, final SVCallRecordWithEvidence b) {
+        if (!isDepthOnlyCall(a) || !isDepthOnlyCall(b)) return false;
+        Utils.validate(a.getContig().equals(a.getEndContig()), "Call A is depth-only but interchromosomal");
+        Utils.validate(b.getContig().equals(b.getEndContig()), "Call B is depth-only but interchromosomal");
+        if (!a.getType().equals(b.getType())) return false;
+        final Set<String> sharedSamples = new LinkedHashSet<>(a.getSamples());
+        sharedSamples.retainAll(b.getSamples());
+        final double sampleOverlap = Math.min(sharedSamples.size() / (double) a.getSamples().size(), sharedSamples.size() / (double) b.getSamples().size());
+        if (sampleOverlap < minSampleOverlap) return false;
+        return getClusteringInterval(a, null)
```

Comment:
Should we be checking the copy number here?  I don't think I want to merge a CN1 deletion with a CN0 deletion, for example.